test.js

```diff
@@ -1,46 +1,63 @@
-import {load, all, exist, getUrl, parse} from './gh-emoji';
+import { load, all, exist, getUrl, parse } from './gh-emoji';
 import tapeTest from 'tape';
 
-function before() {
+const before = () => {};
 
-};
-
-function test(title, cb) {
+const test = (title, cb) => {
   tapeTest(title, (...args) => {
     before();
     cb(...args);
   });
-}
+};
+
+test('all() before load()', assert => {
+  assert.equal(all(), null, 'must return null');
+  assert.end();
+});
 
 test('load', assert => {
   load().then((emojis) => {
-    assert.pass(!!emojis);
+    assert.equal(typeof emojis, 'object', 'must resolve Object with emojis');
     assert.end();
   });
 });
 
-test('all', assert => {
-  assert.pass(!!all());
+test('all() after load()', assert => {
+  assert.equal(typeof all(), 'object', 'must return Object with emojis');
   assert.end();
 });
 
 test('exist', assert => {
-  assert.pass(exist('8ball'));
-  assert.pass(exist('aries'));
-  assert.pass(!exist('fakeEmoji'));
+  assert.ok(exist('8ball'), '8ball emoji exists');
+  assert.ok(exist('aries'), 'aries emoji exists');
+  assert.ok(!exist('fakeEmoji'), 'fakeEmoji emoji does not exists');
   assert.end();
 });
 
 test('getUrl', assert => {
-  assert.pass(getUrl('angel') === 'https://assets-cdn.github.com/images/icons/emoji/unicode/1f47c.png?v6');
-  assert.pass(getUrl('baby') === 'https://assets-cdn.github.com/images/icons/emoji/unicode/1f476.png?v6');
-  assert.pass(!getUrl('fakeEmoji'));
+  assert.ok(getUrl('angel') ===
+    'https://assets-cdn.github.com/images/icons/emoji/unicode/1f47c.png?v6',
+    'angel emoji must return url'
+  );
+  assert.ok(getUrl('baby') ===
+    'https://assets-cdn.github.com/images/icons/emoji/unicode/1f476.png?v6',
+    'baby emoji must return url'
+  );
+  assert.ok(!getUrl('fakeEmoji'), 'fakeEmoji emoji must not return url');
   assert.end();
 });
 
 test('parse', assert => {
-  assert.pass(parse(':bicyclist:') === '<img src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f6b4.png?v6" class="gh-emoji gh-emoji-bicyclist" alt="bicyclist" />');
-  assert.pass(parse('foo :barber: bar :bell:').indexOf("https://assets-cdn.github.com/images/icons/emoji/unicode/1f514.png?v6") > -1);
-  assert.pass(parse('foo :barber: bar :bell:').indexOf("https://assets-cdn.github.com/images/icons/emoji/unicode/1f488.png?v6") > -1);
+  assert.ok(parse(':bicyclist:') === '<img src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f6b4.png?v6" class="gh-emoji gh-emoji-bicyclist" alt="bicyclist" />',
+    'string is properly parsed'
+  );
+  assert.ok(parse('foo :barber: bar :bell:').includes(
+    'https://assets-cdn.github.com/images/icons/emoji/unicode/1f514.png?v6'),
```

Comment:
Here we're testing if parsed string includes this url, this test will break if they change v6 to something else, e.g. v7 :)